packages/cashier-v2/src/lib/CashierOnboarding/components/CashierOnboardingFiatCard/CashierOnboardingFiatCard.tsx

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
+import { useActiveAccount } from '@deriv/api';
 import { onboardingFiatCardIcons } from '../../constants/icons';
 import { CashierOnboardingCard } from '../CashierOnboardingCard';
 import { CashierOnboardingIconMarquee } from '../CashierOnboardingIconMarquee';
 
-const CashierOnboardingFiatCard: React.FC = () => {
+type TProps = {
+    setIsDeposit: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const CashierOnboardingFiatCard: React.FC<TProps> = ({ setIsDeposit }) => {
+    const { data: activeAccount } = useActiveAccount();
+    const isFiat = Boolean(activeAccount?.currency_config?.is_fiat);
```

Comment:
might be out of scope for this PR - but I think we should drift away from conditional operators, 
if active account / currency config is not there, then we probably shouldn't render this component at all.
Ultimately - excessive use of conditojnal chainging is really good at hiding bugs

Reply:
sure @wojciech-deriv 
we can discuss it !